### ISSUE TITLE
Flickering

### ISSUE THREAD

**Issue by psydex** [modbrin/libchanger#2]
Let me show you.
[demo.zip](https://github.com/modbrin/libchanger/files/541759/demo.zip)

**Comment by modbrin** [comment]
Hi, libchanger project is really old now. I didn't know it's still in use somehow. Anyway, it looks like a problem with auto exposure, have you tried manual mode? Secondly, can you describe os, device and source of camera libs. But actually this is issue of L Camera app, not libchanger, so i couldn't do much about it sadly, if not taking sources of L camera and trying to build fixed release.

**Comment by psydex** [comment]
Yes @modbrin , I have tried manual mode, same issue there - it just ignores my setting and brightens up.
I use Stock Nexus 5 with Android 6.0.1 with LCamera and LibChanger installed. 
I've reported the issue to LCamera as well, hopefully they fix it.
